speech/cloud-client/transcribe_onprem.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# Copyright 2020, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+
+# [START speech_transcribe_onprem]
+def transcribe_onprem(local_file_path, api_endpoint):
+    """
+    Transcribe a short audio file using synchronous speech recognition on-prem
+
+    Args:
+      local_file_path: The path to local audio file, e.g. /path/audio.wav
+      api_endpoint: Endpoint to call for speech recognition, e.g. 0.0.0.0:10000
+    """
+    from google.cloud import speech_v1p1beta1
+    from google.cloud.speech_v1p1beta1 import enums
+    import grpc
+    import io
+
+    # api_endpoint = '0.0.0.0:10000'
+    # local_file_path = 'resources/brooklyn_bridge.raw'
+
+    # Set the API endpoint to direct requests to
+    client_options = {"api_endpoint": api_endpoint}
+
+    # Create a gRPC channel to your server
+    channel = grpc.insecure_channel(target=api_endpoint)
+
+    client = speech_v1p1beta1.SpeechClient(
+        client_options=client_options, channel=channel
+    )
```

Comment:
There's no need to pass the `api_endpoint` through client options on the client constructor. Passing a channel is considered "advanced" mode so you're expected to set the endpoint on the channel itself.

Reply:
Thanks for pointing this out, removed `client_options` and just set the endpoint once in the channel